alerts/cluster-network-operator/NoOvnMasterLeader.md

```diff
@@ -0,0 +1,96 @@
+# NoOvnMasterLeader
+
+## Meaning
+
+This alert is triggered when ovn-kubernetes cluster does not have a
+leader for more than 10 minute.
+
+## Impact
+
+When there is no leader, networking control plane is degraded.
+Networking configuration updates applied to the cluster will not be
+implemented while there is no OVN Kubernetes leader.
+Existing workloads should continue to have connectivity.
+OVN-Kubernetes control plane is not functional.
+
+## Diagnosis
+
+### Control plane issue
+
+This can occur multiple control plane nodes are powered off or are unable to
+connect each other via the network. Check that all control plane nodes are
+powered and that network connections between each machine are functional.
+
+    oc get node -l node-role.kubernetes.io/master=""
+
+### Cluster network operator
+
+Cluster Network operator (CNO) manages the CRUD operations for OVN-Kubernetes
+daemonset.
+Verify the CNO is running:
+
+    oc -n openshift-network-operator get pods -l name=network-operator
+
+Verify the [CNO](https://github.com/openshift/cluster-network-operator/) is
+functioning without error by checking the operator Status.
+You can check the Status of the CNO directly using the following command:
+
+    oc describe co network
+
+Pay attention to any error message reported by Status Conditions of type
+Degraded.
+
+Check the CNO logs for when it is reconciling the daemonset for ovnkube-master.
+
+    oc logs deployment/network-operator -n openshift-network-operator
+
+A successful reconcile for this daemonset looks like this in the CNO logs:
+
+    I0228 14:48:30.941130       1 log.go:184] reconciling (apps/v1, Kind=DaemonSet)
+                                openshift-ovn-kubernetes/ovnkube-master
+    I0228 14:48:30.960944       1 log.go:184] update was successful
```

Comment:
I think it makes sense to replace "Control plane issue" and "Cluster network operator" sections with the link to [NoRunningOvnMaster](https://github.com/openshift/runbooks/pull/40) runbook, and add a note to check NoRunningOvnMaster alert before paying attention to NoOvnMasterLeader, since the former one causes the latter.
Then continue with this runbook from "OVN-kubernetes master pods" section only if NoRunningOvnMaster alert is not firing
wdyt?

Reply:
@bmeng You and I discussed this this morning Bo. Add a section after meaning section:

## Fix alerts before continuing 
Check to ensure the following alerts are not firing and resolved before continuing as they may cause this alert to fire:
* NoRunningOvnMaster`